api/server/server.go

```diff
@@ -236,8 +236,10 @@ func NewServer(conf Config) (*Server, error) {
 		grpcServer.Serve(listener)
 	}()
 
+	errc := make(chan error)
 	go func() {
-		grpcWebProxy.ListenAndServe()
+		errc <- grpcWebProxy.ListenAndServe()
+		close(errc)
```

Comment:
I wonder if a channel is used for a particular reason?
Compared to: `if err := grpcWebProxy.ListenAndServe(); err != nil ` and log error.

cc @sanderpick

Reply:
Hmm, good question. I don't remember. Yeah, don't need the channel.